definitions/cross_product.toml

```diff
@@ -0,0 +1,53 @@
+[metrics]
+
+[metrics.mobile_daily_active_users]
+data_source = "mobile_active_users_aggregates"
+select_expression = "SUM(dau)"
+type = "scalar"
+friendly_name = "Mobile DAU"
+description = """
+    This is the official DAU reporting definition. The logic is
+    [defined in `bigquery-etl`](https://github.com/mozilla/bigquery-etl/tree/main/sql_generators/active_users/templates)
+    and is automatically cross-checked, actively monitored, and change controlled.
+    Whenever possible, this is the preferred DAU reporting definition to use for Mobile products.
+    This metric needs to be aggregated by `submission_date`. If it is not aggregated by `submission_date`,
+    it is similar to a "days of use" metric, and not DAU.
+    
+    For more information, refer to [the DAU description in the Mozilla Data Documentation](https://docs.telemetry.mozilla.org/concepts/terminology.html#dau).
+    For questions please contact bochocki@mozilla.com or firefox-kpi@mozilla.com.
+"""
+owner = ["bochocki@mozilla.com", "firefox-kpi@mozilla.com"]
+deprecated = false
+
+[metrics.mobile_dau_kpi]
+data_source = "mobile_active_users_aggregates"
+select_expression = "SUM(IF(FORMAT_DATE('%m-%d', submission_date) BETWEEN '11-18' AND '12-15', dau, 0)) / 28"
+type = "scalar"
+friendly_name = "Mobile DAU KPI"
+description = """
+    The average [Mobile DAU](https://mozilla.github.io/metric-hub/metrics/cross_product/#mobile_daily_active_users) in the 28-day period ending on December 15th. This is the official
+    Mobile DAU KPI reporting definition. The logic for calculating DAU is
+    [defined in `bigquery-etl`](https://github.com/mozilla/bigquery-etl/tree/main/sql_generators/active_users/templates)
+    and is automatically cross-checked, actively monitored, and change controlled.
+    To reconstruct the annual Mobile DAU KPI, this metric needs to be aggregated by
+    `EXTRACT(YEAR FROM submission_date)`.  
+
+    For more information, refer to [the DAU description in the Mozilla Data Documentation](https://docs.telemetry.mozilla.org/concepts/terminology.html#dau).
+    For questions please contact bochocki@mozilla.com or firefox-kpi@mozilla.com.
```

Comment:
I'm currently adding this contact information in the description, but it could be nice if the `owner` field populated in the website.